src/ws/cockpitauth.c

```diff
@@ -307,6 +315,29 @@ type_option (const gchar *type,
   return default_str;
 }
 
+static guint
+timeout_option (const gchar *type)
+{
+  guint timeout = cockpit_ws_auth_process_timeout;
+  guint64 conf_timeout;
+  const gchar* conf = type_option (type, "timeout", NULL);
+
+  if (conf)
+    {
+      conf_timeout = g_ascii_strtoull (conf, NULL, 10);
+      if (errno == ERANGE || errno == EINVAL)
+        timeout = cockpit_ws_auth_process_timeout;
+      else if (conf_timeout > MAX_AUTH_TIMEOUT || conf_timeout > UINT_MAX)
```

Comment:
exceeding the range is a good candidate for a debug warning, in my opinion
While not essential in this case, I think it is good practice to avoid issues that can't be effectively traced.